docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -67,53 +67,145 @@ The package is published in this location:
 
 ## Initialization
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the UID2 service. It also allows you to configure retry intervals in the event of errors.
+- DSPs should create an instance of the BidstreamClient class
+- Sharers should create an instance of the SharingClient class
+- Publishers should create an instance of the Uid2PublisherClient class
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the UID2 service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to UID2, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+You will need to provide the values necessary for the SDK to authenticate with the UID2 service
+
+| Parameter    | Description                                                                               |
+|:-------------|:------------------------------------------------------------------------------------------|
+| `endpoint`   | The endpoint for the UID2 service. See [Environments](../getting-started/gs-environments) |
```

Comment:
Called `base_url`

Reply:
I was trying to sync this with the C# SDK and thus renamed it to endpoint. https://unifiedid.com/docs/sdks/uid2-sdk-ref-csharp-dotnet#initialization